db/queries/get_budget_usage.go

```diff
@@ -0,0 +1,44 @@
+package queries
+
+import (
+	"time"
+
+	"github.com/getAlby/hub/constants"
+	"github.com/getAlby/hub/db"
+	"gorm.io/gorm"
+)
+
+func GetBudgetUsageSat(tx *gorm.DB, appPermission *db.AppPermission) uint64 {
+	var result struct {
+		Sum uint64
+	}
+	tx.
+		Table("transactions").
+		Select("SUM(amount_msat + coalesce(fee_msat, 0) + coalesce(fee_reserve_msat, 0)) as sum").
```

Comment:
why do we need the `coalesce` ? isn't a fee_msat default of 0 ok?

Reply:
Currently fee_msat is nullable, because we don't always know the fee (for some connectors) and it seems wrong for it to be zero (I think we chatted about this in the past)

Reply:
I'd actually add zero in our wallet - imo such exceptions because some connectors will make the app complicated and now we have to do things like `coalesce` because some connector misses a feature. 
and in the next code this `coalesce` or similar will be forgotten and just causes bugs.